pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -159,7 +160,11 @@ func (exp *flowExporter) initFlowExporter(collector net.Addr) error {
 	exp.process = expProcess
 	exp.templateID = expProcess.NewTemplateID()
 
-	expProcess.LoadRegistries()
+	var reg ipfix.IPFIXRegistry
+	reg = ipfix.NewIPFIXRegistry()
```

Comment:
You can move this to NewFlowExporter. We can load the registry there itself. It will be better that way if we add unit test coverage for initFlowExporter in the future.